js/components/controlBar.js

```diff
@@ -248,6 +248,7 @@ var ControlBar = React.createClass({
     } else {
       this.props.controller.toggleMultiAudioScreen();
     }
+    this.handleControlBarFocus(evt);
```

Comment:
I'm remembering just now that we have a big refactor coming up where the `handleControlBarFocus` function will be removed from this component (you can see this update in the PLAYER-3475 branch).  Could we switch to the approach that we use for the closed captions menu? You can see how this was implemented in `closedCaptionsPanel.js`

Reply:
@jpfranco oh,  i am sorry, I did not get your idea about the approach. But anyway we checked CC and the bug is existing and for CC now.

Reply:
Oh ok, it seems to be working for me. Does that only happen when using NVDA?

The problem is that this code has changed a lot, so this focus handler is not even in this file anymore. Please take a look at this PR: https://github.com/ooyala/html5-skin/pull/1083. This logic has been moved to the `PreserveKeyboardFocus` component.

Is it possible to update your fix so that it doesn't rely on the `handleControlBarFocus` function?

Reply:
Note that 1083 has been merged